src/app/modules/profile/components/profile/profile.component.scss

```diff
@@ -0,0 +1,29 @@
+.profile {
+  display: flex;
+  font-family: Montserrat, "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+
+  &-navigation {
+    display: flex;
+    flex-direction: column;
+    width: 30%;
+    height: 97vh;
+    background-color: white;
+    padding: 3vw;
+
+    &-button {
+      width: 60%;
+      border: none;
+      background-color: transparent;
```

Comment:
You can use the appButtonsStyling directive with ButtonType="default" :)

Reply:
I believe it looks different than my buttons on image above🤔